src/utils.ts

```diff
@@ -140,6 +140,29 @@ async function findNodeViaShell(vscode: vscodeTypes.VSCode, cwd: string): Promis
   });
 }
 
+export function addNpmRunPath(env: NodeJS.ProcessEnv, cwd: string): NodeJS.ProcessEnv {
+  const newPath = [];
+
+  let currentPath = path.resolve(cwd);
+  let previousPath;
+  while (previousPath !== currentPath) {
+    newPath.push(path.join(currentPath, 'node_modules', '.bin'));
```

Comment:
Shouldn't we check that `node_modules/.bin` exists? Looks kinda wasteful to add all of them.

Reply:
I'm not sure! My hunch is that the system that resolves `PATH` is *very* optimised and much faster at checking path existence than our code. And because FS operations come at a premium under Windows, leaving the actual checks to the OS would be faster. This is all speculation though - do you know more about this?

Reply:
No, I don't. I see that's what the other project does. Let's keep it as is.